resources/js/components/GroupActions.vue

```diff
@@ -67,6 +91,12 @@ export default {
       })
 
       this.$emit('left')
+    },
+    deleteGroup() {
+      this.$refs.confirmDelete.show()
+    },
+    async deleteConfirmed() {
+      window.location = '/group/delete/' + this.idgroups
```

Comment:
Is it OK that the delete action is a GET?  As it is behind permission checks, probably OK.  Possibly a more philosophical point, could discuss at retro.

Reply:
No, but our use of HTTP verbs is all over the place.  Added to retro.